src/renderer/components/screens/AccountDeletionScreen/AccountDeletionScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react'
+import useTranslationFunction from '../../../hooks/useTranslationFunction'
+
+import Dialog, {
+  DialogBody,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  FooterActionButton,
+  FooterActions,
+} from '../../Dialog'
+
+import styles from './styles.module.scss'
+import { T } from '@deltachat/jsonrpc-client'
+import { BackendRemote } from '../../../backend-com'
+import { Screens } from '../../../ScreenController'
+import { avatarInitial } from '../../Avatar'
+
+export function AccountDeletionScreen({
```

Comment:
You can export them as default if it's the only component

Reply:
<img width="516" alt="Bildschirmfoto 2024-02-09 um 18 29 57" src="https://github.com/deltachat/deltachat-desktop/assets/18725968/bf958262-940c-4045-8b6b-12fa003c768a">
seems harder for vscode to find, but sure I could do that.

Reply:
also you can do default export always, you can still access other exports like with react:
```js
Import React, {useEffect} from"react"
```